warehouse/templates/includes/packaging/project-data.html

```diff
@@ -18,7 +18,7 @@ <h3 class="sidebar-section__title">{% trans %}Verified details{% endtrans %}&nbs
           <small><i>{% trans %}(What is this?){% endtrans %}</i></small>
       </a>
   </h3>
-  <small><i>These details have been verified by PyPI</i></small>
+  <small><i>These details have been verified by PyPI on {{ release.created.date() }}</i></small>
```

Comment:
Language nitpick:

```suggestion
  <small><i>These details were verified by PyPI on {{ release.created.date() }}</i></small>
```

Thinking out loud, though: maybe we should put the verification date behind a tooltip, rather than rendering it in the text here? My thinking is that rendering it adds to the visual noise in this section, but I'm curious what @miketheman and @di think.

Reply:
> have been -> were
fixed!

> text section is missing a {% trans %} group

fixed, and also fixed for the Unverified Details section

Reply:
I agree, this is getting a little busy (especially with the addition of the 'what is this?' bit).

What about a tooltip on the checkmark that says "Verified on {date}"? They would all be the same, but it would make it possible to have re-verifications at different times (or asynchronously after a release is made) in the future.

Reply:
Fixed, I added the date to the existing tooltip of the checkmark, and updated the screenshot and PR description.